### ISSUE TITLE
Lower Accuracy With ResNet BackBone

### ISSUE THREAD

**Issue by tymatfd** [kaixin96/PANet#29]
Hello. I found that when I switched the backbone to resnet with last two layer replaced by dilation2 and dilation4 conv with all other settings unchanged, the accuracy seems much lower than vgg , should I change some hyper paramters , like the learning rate or training steps ? thank you

**Comment by kaixin96** [comment]
Hi @tymatfd , sorry for the late reply. I don't know how much is "much lower", but based on my previous experience, changing the backbone from vgg to resnet50 did not incur performance drop (actually the performance improved a little, about 1~2 percent in mIoU if I remember correctly). I didn't tune the hyper-parameters either. I suggest you debug the code where resnet is plugged and make sure the output feature map is not too small (in vgg, the output feature is 8x smaller than the input image).

**Comment by kaixin96** [comment]
I’m closing this issue because it has been inactive for a while. Feel free to reopen if you have questions. Thank you.
